packages/integrations/src/vue.ts

```diff
@@ -405,3 +378,21 @@ export class Vue implements Integration {
     }
   }
 }
+
+// tslint:disable-next-line: completed-docs
+interface HubType extends Hub {
+  // tslint:disable-next-line: completed-docs
+  getScope?(): Scope | undefined;
+}
+
+/** Grabs active transaction off scope */
+export function getActiveTransaction<T extends Transaction>(hub: HubType): T | undefined {
+  if (hub && hub.getScope) {
+    const scope = hub.getScope() as Scope;
+    if (scope) {
+      return scope.getTransaction() as T | undefined;
+    }
+  }
+
+  return undefined;
+}
```

Comment:
Hmm, I don't like that we have this duplicated everywhere :/

Reply:
Isn't doing `hub.getScope().getTransaction()` the same?

Reply:
`public getScope(): Scope | undefined`

https://github.com/getsentry/sentry-javascript/blob/e8a37d203bb32a8491feed3785d0cc93e567b255/packages/hub/src/hub.ts#L140

Reply:
So it is more like `hub?.getScope?.getTransaction()`

Reply:
We can always refactor this if we figure out a better way to do this.